proto/viam/service/slam/v1/slam.proto

```diff
@@ -67,7 +67,16 @@ message GetPointCloudMapResponse {
   // GetPointCloudMapResponse.point_cloud_pcd_chunk values in the
   // order received result in the complete pointcloud in standard PCD
   // format.
+  //
+  // Read more about the pointcloud format here:
   // https://pointclouds.org/documentation/tutorials/pcd_file_format.html
+  //
+  // Viam expects pointcloud data with fields "x y z" or "x y z rgb", and for
+  // this to be specified in the pointcloud header in the FIELDS entry. If color
+  // data is included in the pointcloud, Viam's services assume that the color
+  // value encodes a confidence score for that data point. Viam expects the
+  // confidence score to be encoded in the blue parameter of the RGB value, on a
+  // scale from 1-100.
```

Comment:
Only question I have is should this description also talk about coordinate system assumptions for maps, but I think if we want to add that here then it should be a part of the documentation changes @jeremyrhyde will make for the coordinate change scope

Reply:
Good point, I'd need some more context on our coordinate system assumptions, but from what I understand that's going to change with Jeremy's work, right? If so, I think we should wait until that's resolved before documenting.

Reply:
Agree, I think it's okay to just document this for now.